app/src/main/java/hu/vmiklos/plees_tracker/MainActivity.kt

```diff
@@ -325,7 +340,17 @@ class MainActivity : AppCompatActivity(), View.OnClickListener {
 
             return
         }
+
         DataModel.start?.let { start ->
+            // When user starts tracking sleep
+            preferences.edit()
+                .putInt("current_dnd", dndManager.currentInterruptionFilter)
+                .apply() // Saves current Do Not Disturb status
+            if (dndManager.isNotificationPolicyAccessGranted and dndEnabled) {
+                dndManager.setInterruptionFilter(NotificationManager.INTERRUPTION_FILTER_PRIORITY)
+            } else {
+                Log.w(TAG, "Failed to enable DND, permissions not enabled")
```

Comment:
Same here, only warn when the setting is on, but we have no access.

Reply:
Ditto